app/src/main/java/org/jellyfin/mobile/player/deviceprofile/DeviceProfileBuilder.kt

```diff
@@ -130,20 +134,43 @@ class DeviceProfileBuilder(
         )
     }
 
+    @RequiresApi(Build.VERSION_CODES.Q)
+    fun canHardwareDecode(codec: String): Boolean {
+        val parsedCodec = when (codec) {
+            "mpeg4" -> "avc1"
```

Comment:
`mpeg4` might not be the same as `h264`, so this line might need to go
```suggestion
```
i weren't really sure about this, there's a lot of conflicting information out there about how `mpeg4` and `h264` relate 🤔